webapp/locales/de.json

```diff
@@ -589,13 +589,18 @@
   },
   "search": {
     "failed": "Nichts gefunden",
+    "for": "Suche nach ",
     "heading": {
-      "Post": "Beiträge",
-      "Tag": "Hashtags",
-      "User": "Benutzer"
+      "Post": "Beitrag ::: Beiträge",
+      "Tag": "Hashtag ::: Hashtags",
+      "User": "Benutzer ::: Benutzer"
     },
     "hint": "Wonach suchst Du?",
-    "placeholder": "Suchen"
+    "no-results": "Keine Ergebnisse für \"{search}\" gefunden. Versuch' es mit einem anderen Begriff!",
+    "page": "Seite",
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-09-23T08:43:12Z" title="Wednesday, September 23rd 2020, 10:43:12 am +02:00">Sep 23, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -590,12 +590,15 @@
   "search": {
     "failed": "Nichts gefunden",
     "heading": {
-      "Post": "Beiträge",
-      "Tag": "Hashtags",
-      "User": "Benutzer"
+      "Post": "Beitrag ::: Beiträge",
+      "Tag": "Hashtag ::: Hashtags",
+      "User": "Benutzer ::: Benutzer"
     },
     "hint": "Wonach suchst Du?",
-    "placeholder": "Suchen"
+    "no-results": "Keine Ergebnisse für \"{search}\" gefunden. Versuch' es mit einem anderen Begriff!",
+    "page": "Seite",
+    "placeholder": "Suchen",
+    "results": "Ergebniss gefunden ::: Ergebnisse gefunden"
```


`Ergebnis` would be correct. 😝